verification/verification.go

```diff
@@ -39,6 +41,18 @@ var TagTCITestCase = TestCase{
 var GetProfileTestCase = TestCase{
 	"GetProfile", TestGetProfile, []string{},
 }
+var InvalidHandleTestCase = TestCase{
+	"CheckInvalidHandle", TestInvalidHandle, []string{"Simulation", "RotateContext", "ExtendTci", "Tagging"},
+}
+var WrongLocalityTestCase = TestCase{
+	"CheckWrongLocality", TestWrongLocality, []string{"AutoInit", "RotateContext", "ExtendTci", "Tagging"},
+}
+var UnsupportedCommand = TestCase{
+	"CheckSupportForCommand", TestUnsupportedCommand, []string{"AutoInit"},
+}
+var UnsupportedCommandFlag = TestCase{
+	"CheckSupportForCommmandFlag", TestUnsupportedCommandFlag, []string{"AutoInit", "RotateContext", "ExtendTci", "Tagging"},
+}
```

Comment:
Can you add all of these to the `AllTestCases` array?

Reply:
**I have some problems with adding.**
1. The AllTestCases has a broad DPE suppport features, so it will not cause/catch errors in  UnsupportedCommand and UnsupportedCommandFlag  testcases, so left as it is.
2. However, I could add the Invalidhandle and Wrong locality test cases under AllTestCases without issues.

Reply:
I don't understand. AllTestCases is just a list if all the test cases. If the target doesn't have the support the test is requesting, the test will be skipped.

Reply:
Adding some more information for context:
1. AllTestCases is list of all testcases that is run with DefaultSupport test target which has more support features.
```
		{
			"DefaultSupport",
			getTestTarget([]string{"AutoInit", "Simulation", "X509", "IsCA", "Tagging", "RotateContext", "ExtendTci"}),
			AllTestCases,
		},
```
2. For **UnSupportedCommand testcase**, we **purposefully exclude** these support features to test for the "UnsupportedCommand" error code - especially **Tagging, RotateContext, ExtendTCI**
4. Similarly, for **UnSupportedCommandFlag testcase**, we **purposefully exclude** these support features to test for the "UnSupportedArgument" error code - especially **Simulation, IsCA, X509, ** 
5. Adding these tests in AllTestCases will ultimately test these two negative tests will Default support test target.
6. **If this happens, some cases in "UnSupportedCommand " and "UnSupportedCommandFlag" will fail as expected error will not be returned.**
7. Here is how it fails
```
    --- FAIL: TestRunAll/DefaultSupport-CheckSupportForCommand (0.00s)
    --- FAIL: TestRunAll/DefaultSupport-CheckSupportForCommmandFlag (0.01s)

```
- It gives many Error messages like so are given when testing ....
```
negativeCases.go:248: [ERROR]: Should return "argument is not supported by this profile, implementation, or integration", but returned no error

```

Reply:
Thanks for the explanation. I have opened https://github.com/chipsalliance/caliptra-dpe/issues/256 to track supporting this. We can merge this PR without this feature though.